index.html

```diff
@@ -0,0 +1,214 @@
+        <nav class="secondary-menu desktop-padding">
+          <ul>
+            <li>
+              <a href="#">
+                <img
+                  src="images/social/small-facebook.svg"
+                  alt="Facebook small icon"
+                />
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img
+                  src="images/social/small-twitter.svg"
+                  alt="Twitter small icon"
+                />
+              </a>
+            </li>
+            <li><a href="#">English</a></li>
+            <li><a href="#">My Page</a></li>
+            <li><a href="#">Logout</a></li>
+          </ul>
+        </nav>
+        <nav class="primary-menu desktop-padding">
+          <a href="/"><img src="images/logo.svg" alt="" /></a>
+          <div class="primary-menu-content">
+            <ul>
+              <li><a href="about.html">About</a></li>
+              <li><a href="#">Program</a></li>
+              <li><a href="#">Join</a></li>
+              <li><a href="#">Register</a></li>
+              <li><a href="#">News</a></li>
+            </ul>
+            <button type="button" class="register-menu-btn orange">
+              Register
+            </button>
+          </div>
+        </nav>
+      </div>
+    </header>
+    <main>
+      <section id="hero" class="desktop-padding">
+        <div class="desktop-padding">
+          <h2 class="orange">"Hello Programmers!"</h2>
+          <div class="hero-title cocogoose">
+            Web development short term training
+          </div>
+          <p class="hero-description">
+            A short term web design and development training held at Arba Minch
+            University. A certificate for taking the training is prepared by the
+            university.
+          </p>
+          <h2 class="hero-date">2022.04.12(TUE)~14(THU)</h2>
+          <span class="hero-address"
+            >@ Twins Building, Arba Minch University</span
+          >
+        </div>
+      </section>
+      <section id="the-training">
+        <div class="training-section desktop-padding">
+          <h2 class="the-training-title the-underline">The Training</h2>
+          <div class="events">
+            <div class="event-item">
+              <img
+                class="event-image"
+                src="./images/icons/lecture-talk.svg"
+                alt="Lecture Talk"
+              />
+              <h2>Lecture</h2>
+              <p>Attend the lectures of the web development course training.</p>
+            </div>
+            <div class="event-item">
+              <img
+                class="event-image"
+                src="./images/icons/class-project.svg"
+                alt="Lecture Talk"
+              />
+              <h2>Projects</h2>
+              <p>
+                Finish class projects to train yourself what you learned in the
+                lecture of the course.
+              </p>
+            </div>
```

Comment:
- [ ] The cards in this section are looking too small and people with eye defects might have to strain to see its details. Please increase it so that it matches the size of the original design.

### Your Design
![image](https://user-images.githubusercontent.com/43995830/157900546-152e0d85-89b6-4960-a4be-1c203e783c9a.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/157900640-1dc41601-9751-48b4-a58b-aeb3e6489593.png)

Reply:
- [ ] [ OPTIONAL ] Great job by implementing the request from the previous review. For now, this section is kinda visible but I think it would be better if you increase again the `font-size` of your text to be more visible.